client/src/includes/messages.test.js

```diff
@@ -0,0 +1,99 @@
+import { initButtonSelects } from './initButtonSelects';
+
+// save our DOM elements to a variable
+const testElements = `
+<div class = "button-select">
+  <input type="hidden" value/>
+  <button class="button-select__option" value>
+    All
+  </button>
+  <button class="button-select__option" value = "true">
```

Comment:
Careful here, looks like you have an extra space.
```suggestion
  <button class="button-select__option" value="true">
```